src/Operator/ThrottleOperator.php

```diff
@@ -44,8 +60,11 @@ function ($x) use ($innerDisp, $observer) {
                     return;
                 }
 
-                $newDisp = Observable::of($x, $this->scheduler)
-                    ->delay($this->nextSend - $now, $this->scheduler)
+                /**
+                 * @var Observable<T> $observable
+                 */
+                $observable = Observable::of($x, $this->scheduler);
+                $newDisp = $observable->delay($this->nextSend - $now, $this->scheduler)
```

Comment:
This one is interesting, while this class accepts the `SchedulerInterface` which can also be the `AsyncSchedulerInterface` however. The `delay` method on an observable only accepts the `AsyncSchedulerInterface`. What to do here?